content/guides/destructuring.adoc

```diff
@@ -211,9 +259,38 @@ When you have nested vectors, you can use `:as` or `&` at any level as well.
 ;= 10 20 [10 20]
 ----
 
+or even:
+
+[source,clojure]
+----
+(def my-multidimensional-line [[1 2 5 0 8] [6 4 3 1 9]])
+(let [[[x0 & coords0 :as orig] [x1 & coords1 :as dest] :as line ] my-multidimensional-line]
```

Comment:
I suspect this line might be too long to fit well on the final page. Maybe think about shorter names or multiple lines.

Reply:
Ok